engine/src/main/java/org/terasology/engine/subsystem/lwjgl/BaseLwjglSubsystem.java

```diff
@@ -38,6 +39,7 @@ public void preInitialise(Context context) {
         if (!initialised) {
             initLogger();
             LWJGLHelper.initNativeLibs();
+            Bullet.init(true);
```

Comment:
The introduction of this bullet-specific line of code in the engine directly breaks the independence of the engine from the physics engine. If calling this method is absolutely necessary, we may have to look at an abstraction for physics engine initialization (like we did with the CollisionShape API). This could probably be done in `PhysicsEngineManager`, as that is what currently sets up bullet as the game physics engine.